mobile/src/components/Card.tsx

```diff
@@ -1,17 +1,29 @@
 import React from "react";
 import { Image } from "react-native";
 
+//fixme: I will probably have to break card up unto a sub-component like the web version and then url will be required
 interface Props {
   url?: string;
+  height?: string;
+  width?: string;
```

Comment:
Why not generalize any style changes and use a single className instead?

Reply:
I could but what if the person using it forgets height or width? This way there is some default value being used

Reply:
You can still have a default width and height and let them override it using tailwind className. Any className that goes later in the string will override any previous one of the same properties

Reply:
To be fair, your approach is not bad or anything. It's just that I prefer to have all the styling be one way and not have props that control basic styling behavior that can be done through className.

Reply:
> To be fair, your approach is not bad or anything. It's just that I prefer to have all the styling be one way and not have props that control basic styling behavior that can be done through className.

Disagree, `className` is _the way_. Just do
```ts
<Image className={`h-96 w-11/12 ${className}`} />
```
The user can just override it by passing height or width properties and they don't have to learn about bespoke props.